src/app/joint-control/joint-control-core/joint-control-core.component.ts

```diff
@@ -9,12 +10,20 @@ import {ActivatedRoute} from "@angular/router";
 })
 export class JointControlCoreComponent implements OnInit {
     joint!: JointConfiguration;
+    motorName: string = "";
```

Comment:
rename to 'selectedMotorName' or something similar, that better indicates the meaning of this variable